### DISCUSSION TITLE
Icons on home screen

### DISCUSSION THREAD

**Discussion by Hooshee** [Blackymas/NSPanel_HA_Blueprint#911] [Q&A] [answered]
Hello Guys,

it has to be my weakness, but I could not find a way to populate that area on the screenshot. What should I set up to get those working?

Thanks

Barna

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/76881082/7959e429-487f-4752-8792-efd370ef8e82)

**Comment by rleongcs** [comment, marked as answer]
Not everything are available right now. They are part of some future features. For now, you only have `Entities`, `QR Code` and `Notification` if I am not mistaken.

**Reply by Blackymas** [reply]
yes it is correct ;) we work on it ;)

**Comment by Hooshee** [comment]
Thanks!

Awesome work btw :)

**Comment by edwardtfn** [comment]
So, now you have all you need to have that many icons on the Home page. 😉
